README.Rmd

```diff
@@ -59,6 +59,46 @@ knitr::kable(head(demo_loanbook))
 I order to run the matching process optimally, here are some useful
 functions to check your data before the matching process.
 
+#### Check crucial columns names
```

Comment:
I think the explanation is great to show the powers of this package. But the intended reader most likely want care about the package -- sad I know :-(. I guess they will read as little as they can get away with -- likely trying to first copy-paste + run the code. So I think all you need it something like this should be enough:

```r
# TODO: Replace `loanbook_demo `with your real loanbook
loanbook <-  loanbook_demo

# Checking your data is as we expect
loanbook |> check_crucial_names()
...
```

If the data doesn't get the right columns, the error message should help them figure it out. Else reword the error message until it's clear.